vitrina/datasets/managers.py

```diff
@@ -3,4 +3,8 @@
 
 class PublicDatasetManager(models.Manager):
     def get_queryset(self):
-        return super().get_queryset()
+        all_datasets = super().get_queryset()
+        public_datasets = all_datasets.filter(is_public__isnull=False).filter(is_public=True)
```

Comment:
```suggestion
        public_datasets = all_datasets.filter(is_public=True)
```

`is_public` is not nullable, so there is no point checking for NULL values.